### DISCUSSION TITLE
CC: Tweaked 1.95.0

### DISCUSSION THREAD

**Discussion by SquidDev** [cc-tweaked/CC-Tweaked#625] [Announcements]
Normally I'd post these release announcements [on the forums][forums], but now discussions are in public beta, I thought it'd be worth giving them a whirl for this too.

Anyway, as is tradition, it's Christmas<sup>1</sup> and so it's time for another CC: Tweaked release. As always, this can be [found on CurseForge][cf]. I'm afraid there's not many exciting changes (although there's lots of bug fixes), but there's a couple of fantastic external contributions I think it's worth covering!

## New help viewer (by @MCJack123)

![The new help viewer](https://user-images.githubusercontent.com/4346137/103139510-de35a200-46d4-11eb-8204-28e855dc2fba.png)

A recent PR by Jack (#595) rewrote the help viewer. Instead of using `textutils.pagedPrint` we now use a new paginator, which supports scrolling up and down, making it much easier to read through documentation.

Of course, the in-game documentation is still a little incomplete. As with everything, there are some plans to improve that (#568).

## New `cc.strings` module (@Lupus590)
Lupus added a new [`cc.strings`](https://tweaked.cc/module/cc.strings.html) module, which builds on the existing `textutils` library.

There's not oodles in there right now, but it _does_ have a function for wrapping text (much like `print` does), which I'm sure will come in handy. I know it's already proved its worth in the ROM!

---

Many thanks to the many people who contributed towards this release, both in raising bugs and contributing code!

That's all (hopefully) for me this year. Have a fantastic Christmas, and a lovely new year!

---

<sup>1. ish. Previous Christmas releases have been on the 20th (2018) and 24th (2019) instead.<sup>

[forums]: http://www.computercraft.info/forums2/index.php?/topic/29161-mc112fork-cc-tweaked "CC:T on the CC forums"
[cf]: https://www.curseforge.com/minecraft/mc-mods/cc-tweaked/files "CC:T on CurseForge"

**Comment by Lupus590** [comment]
I wonder if I should refactor ROM so that it uses cc.strings for word wrapping in places where ROM rolls its own. I think it might only be bios which will be a pain as require is not useable there.

**Comment by Lupus590** [comment]
Also, it's Christmas, go have fun. :P

**Comment by Lupus590** [comment]
Is there no 1.16 release of 1.95?

**Reply by SquidDev** [reply]
Weird, I could have sworn I can the release script, but apparently not. Will do that in the morning.
